### PR TITLE
Examples were flipped

### PR DESCRIPTION
**Pull request by AvrumFeldman**
I didn't test if my changes are correct per se. I just fixed the examples to match their notation.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/policy-and-compliance/ediscovery/message-properties-and-search-operators.md [hunk 1]
```diff
@@ -97,8 +97,8 @@ Here's a list of the unsupported characters for an In-Place eDiscovery search qu
 
 - When searching a recipient property, such as To, From, Cc, or Recipients, you can use an SMTP address, alias, or display name to denote a recipient. For example, you can use pilarp@contoso.com, pilarp, or "Pilar Pinilla".
 
-- You can use only prefix wildcard searches (for example, **cat\*** or **set\***). Suffix wildcard searches (\*cat) or substring wildcard searches (\*cat\*) aren't supported.
+- You can use only prefix wildcard searches (for example, **\*cat** or **\*set**). Suffix wildcard searches (cat\*) or substring wildcard searches (\*cat\*) aren't supported.
 
 - When searching a property, use double quotation marks (" ") if the search value consists of multiple words. For example **subject:budget Q1** returns messages that contain **budget** in the in the subject line and that contain **Q1** anywhere in the message or in any of the message properties. Using **subject:"budget Q1"** returns all messages that contain **budget Q1** anywhere in the subject line.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/policy-and-compliance/ediscovery/message-properties-and-search-operators.md:102]:
```suggestion
- When searching a property, use double quotation marks (" ") if the search value consists of multiple words. For example **subject:budget Q1** returns messages that contain **budget** in the subject line and that contain **Q1** anywhere in the message or in any of the message properties. Using **subject:"budget Q1"** returns all messages that contain **budget Q1** anywhere in the subject line.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AvrumFeldman, not related to this PR but please commit the additional typo correction. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**